ui/state.go

```diff
@@ -24,26 +24,24 @@ func (win *Window) updateStates(update interface{}) {
 
 		if e.LoadedWallets > 0 {
 			// set wallets and accounts tab when wallet info is updated
-			go func() {
-				wallets := make([]decredmaterial.TabItem, len(e.Wallets))
-				for i := range e.Wallets {
-					wallets[i] = decredmaterial.TabItem{
-						Title: e.Wallets[i].Name,
-					}
+			wallets := make([]decredmaterial.TabItem, len(e.Wallets))
+			for i := range e.Wallets {
+				wallets[i] = decredmaterial.TabItem{
+					Title: e.Wallets[i].Name,
 				}
-				win.walletTabs.SetTabs(wallets)
+			}
+			win.walletTabs.SetTabs(wallets)
 
-				accounts := make([]decredmaterial.TabItem, len(e.Wallets[win.selected].Accounts))
-				for i, account := range e.Wallets[win.selected].Accounts {
-					if account.Name == "imported" {
-						continue
-					}
-					accounts[i] = decredmaterial.TabItem{
-						Title: e.Wallets[win.selected].Accounts[i].Name,
-					}
+			accounts := make([]decredmaterial.TabItem, len(e.Wallets[win.selected].Accounts))
+			for i, account := range e.Wallets[win.selected].Accounts {
+				if account.Name == "imported" {
+					continue
 				}
-				win.accountTabs.SetTabs(accounts)
-			}()
+				accounts[i] = decredmaterial.TabItem{
+					Title: e.Wallets[win.selected].Accounts[i].Name,
+				}
+			}
+			win.accountTabs.SetTabs(accounts)
 		}
```

Comment:
The win.walletTabs.SetTabs(wallets) is not really used anymore in the project expect in the deprecated.go file. since this is a PR for code clean up, i believe it time to remove the deprecated.go file as non of the methods there are used in any part of the project.
These lines of code can also be removed.